kube/helm/helm.go

```diff
@@ -113,6 +105,10 @@ func (hc *Actions) Get(name string) (*release.Release, error) {
 
 // ListReleases lists chart releases
 func (hc *Actions) ListReleases() ([]compose.Stack, error) {
+	err := hc.initialize(nil)
+	if err != nil {
+		return nil, err
+	}
```

Comment:
This code should be removed alltogether ?

Reply:
Need to try to improve this required initialize call setting up the logger. will do in a follow up PR